### PR TITLE
Fabo/polkadot fees

### PR DESCRIPTION
**Pull request by faboweb**
Closes #ISSUE

**Description:**

Thank you! 🚀

---

For contributor:

- [ ] Added changes entries. Run `yarn changelog` for a guided process.
- [ ] Reviewed `Files changed` in the github PR explorer
- [ ] Attach screenshots of the UI components on the PR description (if applicable)
- [ ] Scope of work approved for big PRs

For reviewer:

- [ ] Manually tested the changes on the UI

### CODE REVIEW

**Comment by Bitcoinera** [comment]
It works! (just takes a while)



Amazing! Not at all a draft PR

**Comment by Bitcoinera** [comment]
Well, actually we should change the next button for a loader or something like that... just did a quick fix

File: src/ActionModal/components/ActionModal.vue [hunk 1]
```diff
@@ -247,7 +247,9 @@
                 type="primary"
                 value="Next"
                 :disabled="
-                  disabled || (step === feeStep && $v.invoiceTotal.$invalid)
+                  disabled ||
+                    (step === feeStep && $v.invoiceTotal.$invalid) ||
+                    (step === feeStep && !estimatedFee)
```
Comment (faboweb) [inline src/ActionModal/components/ActionModal.vue:252]: doesn't work. if estimated fee is 0 this will always block.

Comment (faboweb) [reply src/ActionModal/components/ActionModal.vue:252]:
```suggestion
                    (step === feeStep && !gasEstimateLoaded)
```

Comment (Bitcoinera) [reply src/ActionModal/components/ActionModal.vue:252]: oh true!

**Comment by faboweb** [comment]
getPolkadotFees is hard to test because it is actually querying the Polkadot API

**Comment by Bitcoinera** [comment]
What do you think about this "Fetching fees..." message I just added? I think it is better so users understand what is going on:


Lunie-Simple-Staking-Wallet-Stak

**Comment by Bitcoinera** [comment]
The loader somehow doesn't work very well with Polkadot. It gets stuck from time to time... Not the best

**Comment by faboweb** [comment]
@jbibla any opinion?

**Comment by jbibla** [comment]
why don't we hardcode a value like always?

**Comment by faboweb** [comment]
I have no idea yet about the Polkadot fees and how dynamic they are and what good values are.

**Comment by jbibla** [comment]
can we make the request for fees on the first step or do we have to wait for the second step?

**Comment by faboweb** [comment]
> can we make the request for fees on the first step or do we have to wait for the second step?

easier to do as is. as we already know the properties of the request which you need for creating the transaction object to do the fees calculation. For Cosmos we could do the request earlier. But I decided to defer to improve speed of loading the actionModel initially.

**Comment by faboweb** [comment]
Needs to be added in the extension as well

**Comment by faboweb** [comment]
I confirmed that we need the transaction details for the fee calculation. We can connect to the API immediately though, which could speed up things.

**Comment by Bitcoinera** [comment]
We could have it as a GraphQL query, is this what you have in mind? `polkadotFees(...transactionDetails)`

**Comment by faboweb** [comment]
I didn't but we could do that as well.

**Pull request merged**